app/controllers/submissions_controller.rb

```diff
@@ -34,6 +34,7 @@ class SubmissionsController < ApplicationController
                        :update_files,
                        :populate_file_manager_react]
   before_filter :authorize_for_user, only: [:download, :downloads]
+  layout "assignment_content", only: [:browse]
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.